### PR TITLE
Added method to refresh skills page and passed as prop to modal

### LINKED ISSUE #133: If there is a blog-type link, render the most recent RSS feed of that link
This tool might work.  http://matthewcodes.github.io/blog/2015/04/18/react-dot-js-feedwidget/

This could work too:
https://feed.mikle.com/?gclid=CMCOjPbrhKUCFQNY2godQ14IPw

### PR DESCRIPTION
**Pull request by dane-johnson**
Fixes #133

### CODE REVIEW

File: src/Skills.jsx [hunk 1]
```diff
@@ -91,6 +92,14 @@ class Skills extends Component {
      });
   }
 
+  reloadSkills() {
```
Comment (thebho) [inline src/Skills.jsx:109]: If this has the same functionality as componentDidMount(), you should have that method call this too.  No reason to call the same axios.get() in 2 different places

**Review by thebho — CHANGES_REQUESTED** [review]

**Comment by solkaz** [comment]
Other than Brian's requested changes, this looks good to me, and it implements most of the changes I had made. I had gone further and refactored SkillForm to use a single callback to Skills to handle getting the new skill's name and type and closing the modal, but I can implement those changes on top of this PR

**Review by dane-johnson — COMMENTED** [review]
This all LGTM but I think the purpose of this issue was really just to fix the not refreshing bug, whereas this seems like a major refactor of the skills file. I do think that is something we need, but I don't know if that should really be done here.

**Review by Hopding — APPROVED** [review]
Nice 👍

**Comment by dane-johnson** [comment]
I think this is good to go

**Review by thebho — APPROVED** [review]

**Comment by thebho** [comment]
Looks good, will merge in morning

**Pull request merged**